### ISSUE TITLE
Hive in the Tower (1126) (BB #898)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#898]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 23.05.2015 13:29:07 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/898

<hr>

You are unable to click on "Hive'Ashi Pod" so u are unable to continue the quest.

![WoWScrnShot_052315_152758.jpg](https://bitbucket.org/repo/kMeMB/images/77788615-WoWScrnShot_052315_152758.jpg)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 28.05.2015 15:14:12 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/898/#comment-18432271

<hr>

Confirmed

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 28.05.2015 19:42:37 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/898/#comment-18437451

<hr>

Fix #898, #912 & #909

→ <<cset 662f81fc79f4>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.05.2015 01:22:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/898/#comment-18473901

<hr>

[Core/Opcodes] Fix a sanity check in HandleGameObjectUseOpcode that was added to
               block certain exploits. As far as I'm aware, the fix is still valid
               but now we have returned a majority of broken gameobjects to normal.

```
           Fixes #931, #943.
           Correctly fixes #912, #909 and #898.
```

→ <<cset e215d2efab6f>>
